simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,33 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(binaryList: List[int]) -> List[int]:
+    lenBinaryList = len(binaryList)
+    
+    if lenBinaryList == 0:
+        return -1
+    
+    for digit in binaryList:
+        if digit != 0 and digit != 1:
+            return -1
+
+    digit = 0
+    binaryListIncrement = []
+    carry = 1
+    while digit < lenBinaryList:
+        tempSum = binaryList[digit] + carry
+        if tempSum >> 1:
+            binaryListIncrement.append(0)
```

Comment:
Are you sure we always append(0) here? is there any different between tempSum = 2 and tempSum = 3?